mne/beamformer/tests/test_ap.py

```diff
@@ -0,0 +1,112 @@
+# Authors: Yuval Realpe <yuval.realpe@gmail.com>
+#
+# License: BSD-3-Clause
+
+import os.path as op
+
+from random import choice
+
+import mne
+from mne.beamformer import alternating_projections
+from mne.cov import regularize
+from mne.datasets import testing
+from mne.beamformer.tests.test_rap_music import simu_data, _check_dipoles
+
+data_path = testing.data_path(download=False)
+fname_ave = op.join(data_path, 'MEG', 'sample', 'sample_audvis-ave.fif')
+fname_cov = op.join(data_path, 'MEG', 'sample', 'sample_audvis-cov.fif')
+fname_fwd = op.join(data_path, 'MEG', 'sample',
+                    'sample_audvis_trunc-meg-eeg-oct-4-fwd.fif')
+
+
+def _get_data(ch_decim=1):
```

Comment:
why not reusing the _get_data function in test_rap_music? eventually adding a parameter to avoid copy-pasting most of the function

Reply:
I've decided to import _get_data() from test_rap.py which doesn't use randomness in test and otherwise is (almost) the same.